templates/operator.yaml.j2

```diff
@@ -86,6 +86,13 @@ spec:
             spec:
               type: object
               properties:
+                disperse:
```

Comment:
should we change the version of CRD for this?

Reply:
We can change. But not breaking the previously supported ones

Reply:
ok!